projects/kubernetes/kubernetes/1-20/patches/0008-EKS-PATCH-additional-subnet-configuration-for-AWS-EL.patch

```diff
@@ -0,0 +1,695 @@
+From 039fbec8dd1d9d496ba86de1a0047dcd45d5e53f Mon Sep 17 00:00:00 2001
+From: Jyoti Mahapatra <jyotima@amazon.com>
+Date: Tue, 9 Mar 2021 05:43:04 +0000
+Subject: [PATCH] --EKS-PATCH-- additional subnet configuration for AWS ELB
+
```

Comment:
To keep consistent with how [this patch was in the last release](https://github.com/kschumy/eks-distro/blob/main/projects/kubernetes/kubernetes/1-19/patches/0010-EKS-PATCH-additional-subnet-configuration-for-AWS-el.patch#L6-L9), can we add the following as a description? 

```
* support subnets without cluster tag for auto-discovery
* add support for manual subnet configuration via service annotation

ref: https://github.com/kubernetes/kubernetes/pull/97431
```

Reply:
done